(cleanup) Update error-handling.md

Rudimental opened the pull request:
Minor grammar change from a semi-colon to a period (after the word 'calls').
Now = 'You define error-handling middleware last, after other `app.use()` and routes calls. For example:'
Was = 'You define error-handling middleware last, after other `app.use()` and routes calls; For example:'

Rudimental committed:
(cleanup) Update error-handling.md

Minor grammar change from a semi-colon to a period (after the word 'calls').
Now = 'You define error-handling middleware last, after other `app.use()` and routes calls. For example:'
Was = 'You define error-handling middleware last, after other `app.use()` and routes calls; For example:'

crandmck commented:
Thanks for the suggestion.

In this case, a semicolon is appropriate since the two clauses are closely related.  Either semicolon or period is correct, but I believe a semicolon is better in this case.  For general guidelines on use of semicolons, see (for example):
- https://writing.wisc.edu/Handbook/Semicolons.html
- http://grammar.ccc.commnet.edu/grammar/marks/semicolon.htm
- http://theoatmeal.com/comics/semicolon

However, the "F" should not be capitalized.  If you will change period back to semicolon and lowercase the "f", I will merge the PR.

Rudimental commented:
You're welcome! Sure, I can do that.

Rudimental committed:
Update error-handling.md

Minor capitalization change for word after semicolon.

Now = 'You define error-handling middleware last, after other app.use() and routes calls; for example:'
Was = 'You define error-handling middleware last, after other app.use() and routes calls; For example:'

crandmck commented:
Thanks again!
